docs/theme_architecture/templates.md

```diff
@@ -60,6 +60,15 @@ The selection is passed via a single `item` [Modifier Selection]({% link docs/re
 
 Each selection page is automatically generated at `site.com/items/*booking-id*/selections` and can only be accessed by the customer to whose booking the selection applies.
 
+## Recommendation
+`Optional`
+
+The template is passed via a single `recommendation` [Recommendation]({% link docs/reference/objects/recommendation.md %}) object.
+
+The recommendation page is automatically generated at `site.com/recommendations/*recommendation_slug*`.
+This template will only render if the template is present, otherwise the default recommendation page will be rendered.
+It will also only render if the recommendation has an admin created booking attached (The new recommendation system using booking edits).
```

Comment:
Wonder whether we expose this last bit but perhaps it's useful for now, as these docs are mostly only used internally.

Nit: Can you use lowercase "T"? 

> ... admin created booking attached (the new recommendation system using booking edits).

Reply:
Yeah, was in two minds about it we could just tell @jessehigson and @samantha-lind how it works and leave this out of the docs

Reply:
Can we simplify it and say something like:

The recommendation page is automatically generated at `site.com/recommendations/*recommendation_slug*` when a valid recommendation has been created.

This template will only render if it has been configured on the associated site - otherwise, the default recommendation page will be rendered.

Reply:
Wonderfully worded 🤩